### PR TITLE
CEM-1012 List

### PR DESCRIPTION
**Pull request by Ampatte2**

### CODE REVIEW

**Comment by Ampatte2** [comment]
Added list component, fixed output directory should solve build issues.

File: src/Containers/List/List.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    footer?: React.ReactElement;
+    id: string;
+    columnWidth?: string;
+    margin?: string;
+    right?: string;
+    left?: string;
+    translateX?: string;
+    position?: string;
+    isToggleable?: boolean;
+    isLeftToggle?: boolean;
```
Comment (ralph-dev) [inline src/Containers/List/List.tsx:20]: What does this mean? Put descriptions if the prop name is not clear.

**Review by ralph-dev — APPROVED** [review]
Lgtm but one small issue, naming is unclear or prop needs comment

**Review by ralph-dev — APPROVED** [review]
Lgtm

**Pull request merged**